int/hwsc-file-metadata-svc/hwsc-file-metadata-svc.js

```diff
@@ -25,6 +30,11 @@ function getStatus(callback) {
 }
 
 function createFileMetadata(fileMetadata, callback) {
+    if (typeof callback !== "function") {
+        console.error("callback not a function");
+        return;
+    }
+
     const client = new hwscFileMetadataSvcPbJs.FileMetadataService("localhost:50051",
```

Comment:
const for localhost:50051?

Reply:
or function that returns new client connection?

```
function createClient() {
   return new hwscFileMetadataSvcPbJs.FileMetadataService("localhost:50051",
        grpc.credentials.createInsecure());
}
```

Reply:
I also see this a lot too:
```javascript
function (err, response) {
        if (!err) {
            console.log(request);
            grpc.closeClient(client);
        }

        callback(err, response);
    });
```

Will this work?

```javascript
client.createFileMetadata(request, (err, response) => processResponse(err, response, callback);

function processResponse(err, response, callback) {
    if (!err) {
      console.log(request);
      grpc.closeClient(client);
    }

    callback(err, response);
}
```

Reply:
> const for localhost:50051?

we will start a ticket for config file

Reply:
> and function that returns new client connection? Will that even work?
> 
> ```js
> function createClient() {
>    return new hwscFileMetadataSvcPbJs.FileMetadataService("localhost:50051",
>         grpc.credentials.createInsecure());
> }
> ```

we will start a ticket